src/PKSim.Core/CoreConstants.cs

```diff
@@ -46,6 +46,7 @@ public static class Filter
          public const string USER_TEMPLATE_DB_EXTENSION = ".templateDbUser";
          public const string SYSTEM_TEMPLATE_DB_EXTENSION = ".templateDbSystem";
          public const string GENE_DB_EXTENSION = ".expressionDb";
+         public static readonly string MARDOWN_EXTENSION = ".md";
```

Comment:
Mar**k**down

Reply:
going to fix shortly

Reply:
done. This should probably be move to Core anyways. We'll make a note of that and do it when movign sutff to core